toc.yml

```
@@ -42,5 +42,7 @@
     topicHref: articles/enablement/enbl-ref-services.md
   - name: UKCloud Portal
     topicHref: articles/portal/ptl-gs.md
+  - name: IDAM
```

May be worth calling this

```suggestion
  - name: ID and Access Management API
```

IDAM doesn't tell you anything if you don't already know what it is. API helps avoid disappointment if you were expecting a UI.

---

Resolved in bcf5ffa, along with other references to IDAM.